addons/l10n_ec/models/account_chart_template.py

```diff
@@ -7,6 +7,12 @@
 class AccountChartTemplate(models.Model):
     _inherit = "account.chart.template"
 
+    def _get_account_from_template(self, company, template):
+        if template:
+            account = self.env['account.account'].search([('company_id', '=', company.id), ('code', '=', template.code)])
+            return account
+        return self.env['account.account']
+
```

Comment:
Had to do this because we don't have access to `acc_template_ref` like in `_prepare_all_journal`, there will be a proper way in 17.0